PKGBUILD

```diff
@@ -0,0 +1,50 @@
+# Note that you'll need to supply the UE1 games you want to play yourself
+# And need a Vulkan capable GPU to run SurrealEngine
+
+pkgname=surrealengine-git
+pkgdesc="A reimplementation of Unreal Engine 1 (Git version)"
+pkgver=0.0.1
+pkgrel=1
+arch=('x86_64')
+depends=('libx11' 'spirv-tools' 'vulkan-icd-loader' 'vulkan-driver' 'sdl2' 'alsa-lib')
+makedepends=('gcc' 'git' 'cmake' 'vulkan-driver' 'sdl2' 'alsa-lib' 'waylandpp')
+optdepends=('waylandpp: For ZWidget Wayland backend')
+provides=('surrealengine')
+url="https://github.com/dpjudas/SurrealEngine"
+license=('BSD-3-Clause AND MIT AND various others')
+source=("git+${url}.git")
+sha512sums=('SKIP')
+
+build() {
+  cd SurrealEngine
+  [[ -d build ]] && rm -rf build
+  mkdir build && cd build
+  cmake -DCMAKE_BUILD_TYPE=Release ..
+
+  make -j 16
+}
+
+package() {
```

Comment:
Don't forget to include the relevant license files.

Reply:
Now the LICENSE and README gets included in the package too! Thanks for the heads up!